Mathlib/Algebra/GroupPower/Basic.lean

```diff
@@ -476,6 +476,15 @@ lemma mul_zpow_self (a : G) (n : ℤ) : a ^ n * a = a ^ (n + 1) := (zpow_add_one
 #align zpow_mul_comm zpow_mul_comm
 #align zsmul_add_comm zsmul_add_comm
 
+theorem zpow_eq_zpow_zmod {x : G} (m : ℤ) {n : ℤ} (h : x ^ n = 1) :
```

Comment:
I'm not sure if zmod is the right name for this, but I don't have any other suggestions either.

Reply:
This `%` is [Int.mod](https://leanprover-community.github.io/mathlib4_docs/Init/Data/Int/Basic.html#Int.mod) right? In this case the convention seems to just use `mod`.

Reply:
Oh thanks! In which case maybe I'll go for `mod` and `mod'`.